apis/fluentbit/v1alpha2/clusterinput_types.go

```diff
@@ -65,7 +65,7 @@ func (a InputByName) Less(i, j int) bool { return a[i].Name < a[j].Name }
 
 // +kubebuilder:object:root=true
 
-// ClusterInputList contains a list of Input
+// InputList contains a list of ClusterInput
 type InputList struct {
```

Comment:
List type is part of the CRD, the name should be changed to ClusterInputList, same for other CRDs